### PR TITLE
Bugfix/file sorting

### PR DESCRIPTION
**Pull request by ubidefeo**
fixed file sorting for both board and disk

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,13 @@ function listFolder(folder) {
     let filePath = path.resolve(folder, f)
     return !fs.lstatSync(filePath).isDirectory()
   })
+
```
Comment (murilopolese) [inline index.js:22]: The old code is still commented.

File: ui/arduino2/store.js [hunk 2]
```diff
@@ -688,11 +694,15 @@ async function getAvailablePorts() {
 
 async function getBoardFiles(path) {
   await serial.get_prompt()
-  const files = await serial.ilistFiles(path)
-  return files.map(f => ({
+  let files = await serial.ilistFiles(path)
+  console.log(files)
```
Comment (murilopolese) [inline ui/arduino2/store.js:698]: Console log

File: ui/arduino2/store.js [hunk 3]
```diff
@@ -671,11 +671,17 @@ async function selectDiskFolder() {
...
+  let files = await disk.ilistFiles(path)
+  files = files.map(f => ({
     fileName: f.path,
     type: f.type
   }))
+  files = files.sort(filesSortAlphabetically)
+  return files
+}
+
+function filesSortAlphabetically(entryA, entryB) {
```
Comment (murilopolese) [inline ui/arduino2/store.js:683]: I would like to propose the function name to be `sortFilesAlphabetically` to more or less follow the verb-noun names of other methods.

**Review by murilopolese — COMMENTED** [review]
There is some commented code and console log left behind.
I also made a suggestion on the sorting function name.

**Comment by ubidefeo** [comment]
will update and PR again :)

**Comment by ubidefeo** [comment]
@murilopolese I have fixed the PR

**Review by murilopolese — APPROVED** [review]

**Pull request merged**